Elevel/Application/Features/ReportCommands/GetReportDetailQuery.cs

```diff
@@ -34,7 +35,15 @@ public Handler(IApplicationDbContext context, IMapper mapper, UserManager<User>
 
             public async Task<Response> Handle(Request request, CancellationToken cancellationToken)
             {
-                var report = await _context.Reports.AsNoTracking()
+                if (request == null)
+                {
+                    throw new NotFoundException(nameof(request));
+                }
+
+                var report = await _context.Reports
+                    .AsNoTracking()
+                    .Include(x=>x.Creator)
```

Comment:
you can use ProjectTo instead of mapping later.

Reply:
Done